ingest/config.py

```diff
@@ -34,8 +35,12 @@ def get_metric_properties():
 class MetricProperties:
     """Sets default properties for Mixpanel events"""
 
-    def __init__(self, study, study_file):
+    USER_ID = "2f30ec50-a04d-4d43-8fd1-b136a2045079"
+
+    def __init__(self, study, study_file, user_uuid=None):
+        distinct_id = user_uuid if user_uuid else MetricProperties.USER_ID
```

Comment:
Rails refers to this `user_uuid` as `metrics_uuid`.  Let's do similarly:

```suggestion
    def __init__(self, study, study_file, metrics_uuid=None):
        distinct_id = metrics_uuid if metrics_uuid else MetricProperties.USER_ID
```

(The slightly expanded form `user_metrics_uuid` seems OK, but if shortening it's worth being consistent with Rails.)